web-common-marlowe/src/Marlowe/Extended.purs

```diff
@@ -29,15 +29,45 @@ type ContractTemplate
     }
 
 type MetaData
-  = { contractName :: String
-    , contractType :: String
+  = { contractType :: ContractType
     , contractDescription :: String
     , roleDescriptions :: Map S.TokenName String
     , slotParameterDescriptions :: Map String String
     , valueParameterDescriptions :: Map String String
     , choiceDescriptions :: Map String String
     }
 
+data ContractType
+  = ES
```

Comment:
I'd favour long names here. We are going to use this in code, so having ES, EC and having to remember which one was which is painful. I'd rename to 

```
data ContractType 
  = Escrow 
  | EscrowWithCollatoral
  | ...
```